subsys/mgmt/mcumgr/smp_bt.c

```diff
@@ -191,10 +297,67 @@ int smp_bt_unregister(void)
 	return bt_gatt_service_unregister(&smp_bt_svc);
 }
 
+/* BT connected callback. */
+static void connected(struct bt_conn *conn, uint8_t err)
+{
+	alloc_conn_param_data(conn);
+}
+
+/* BT disconnected callback. */
+static void disconnected(struct bt_conn *conn, uint8_t reason)
+{
+	struct conn_param_data *cpd = get_conn_param_data(conn);
+
+	if (cpd->latency_state) {
+		/* Cancel work if ongoing. */
+		(void)k_work_cancel_delayable(&cpd->dwork);
+	}
+
+	/* Clear cpd. */
+	cpd->latency_state = 0;
+	cpd->conn = NULL;
+}
+
+/* BT LE connection parameters updated callback. */
+static void le_param_updated(struct bt_conn *conn, uint16_t interval,
+			     uint16_t latency, uint16_t timeout)
+{
+	struct conn_param_data *cpd = get_conn_param_data(conn);
+
+	if (cpd->latency_state != 0) {
```

Comment:
Maybe we should mark that low latency was already enabled anyway (also in case we did not request it)?

Reply:
You could also check that on connected callback